api.go

```diff
@@ -495,6 +495,8 @@ func (c Client) executeMethod(method string, metadata requestMetadata) (res *htt
 
 		// Read the body to be saved later.
 		errBodyBytes, err := ioutil.ReadAll(res.Body)
+		// res.Body should be closed
+		res.Body.Close()
```

Comment:
you can use "closeResponse()" itself. there is a need to do that in Go http client.